src/index.js

```diff
@@ -137,41 +121,69 @@ function installTimers(pluginConfig, lambdaContext) {
 	const memoryLimit = lambdaContext.memoryLimitInMB;
 
 	function timeoutWarningFunc(cb) {
-		const Raven = pluginConfig.ravenClient;
-		ravenInstalled && Raven.captureMessage("Function Execution Time Warning", {
-			level: "warning",
-			extra: {
-				TimeRemainingInMsec: lambdaContext.getRemainingTimeInMillis()
+		const Sentry = pluginConfig.sentryClient;
+		if (sentryInstalled) {
+			Sentry.captureMessage("Function Execution Time Warning", {
+				level: "warning",
+				extra: {
+					TimeRemainingInMsec: lambdaContext.getRemainingTimeInMillis()
+				}
+			});
+			const client = Sentry.getCurrentHub().getClient();
+			if (client) {
+				client.flush(1000).then(function() {
```

Comment:
This seems new and could use some comments explaining what it actually does

Reply:
captureMessage & captureException are no promises nor do they provide a callback when finished with the transfer to sentry.io. This tells the Hub client to flush all open events and wait max 1000ms or return when finished.

Reply:
found a newer/shorter version of flush :-)